todo/tests/unit/views/test_task.py

```
@@ -0,0 +1,59 @@
+from unittest import TestCase
+from rest_framework.test import APIRequestFactory
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
+
+from todo.views.task import TaskView
+from todo.constants.task import DEFAULT_PAGE_LIMIT
```

Ruff (0.8.2)

`todo.constants.task.DEFAULT_PAGE_LIMIT` imported but unused

Remove unused import: `todo.constants.task.DEFAULT_PAGE_LIMIT`

(F401)